hazelcast-client/src/main/java/com/hazelcast/client/impl/DefaultClientExtension.java

```diff
@@ -40,7 +40,7 @@
 import com.hazelcast.map.impl.MapService;
 import com.hazelcast.nio.ClassLoaderUtil;
 import com.hazelcast.nio.SocketInterceptor;
-import com.hazelcast.nio.tcp.PlainChannelFactory;
+import com.hazelcast.internal.networking.PlainChannelFactory;
```

Comment:
Moved PlainNioChannelFactory to com.hazelcast.internal.networking since both client and server use these classes.